lib/src/main/java/com/shopify/checkoutsheetkit/CheckoutWebView.kt

```diff
@@ -247,19 +247,32 @@ internal class CheckoutWebView(context: Context, attributeSet: AttributeSet? = n
             }
         }
 
+        private fun isRecoverable(statusCode: Int): Boolean {
+            return when (statusCode) {
+                TOO_MANY_REQUESTS -> true
+                in CLIENT_ERROR -> false
```

Comment:
Probably worth checking this one.. Maybe client errors could also be resolved in an external view